src/libLookup/Lookup.cpp

```diff
@@ -3015,14 +3026,26 @@ bool Lookup::ProcessSetTxBlockFromSeed(
         m_syncType = SyncType::NO_SYNC;
         this_thread::sleep_for(chrono::seconds(NEW_NODE_SYNC_INTERVAL));
         m_mediator.m_ds->RejoinAsDS(false);
+      } else if (m_syncType == SyncType::NEW_LOOKUP_SYNC) {
+        m_rejoinInProgress = true;
+        cv_setRejoinRecovery.notify_all();
+        RejoinNetwork();
       }
       return false;
+    } else if (latestSynBlockNum > lowBlockNum) {
+      LOG_GENERAL(INFO,
+                  "Seems we have received all or few of txblocks "
+                  "previously. "
+                  "so ignoring these txblocks!");
+      // cv_setRejoinRecovery.notify_all(); //TODO : Check with @Sandip if we
```

Comment:
need to notify right? otherwise `cv_setRejoinRecovery` will wait for 180 sec in `InitSync`

Reply:
Done